### ISSUE TITLE
Limiting who sees reports.

### ISSUE THREAD

**Issue by ncberman96** [gravitylow/AntiCheat#122]
Hello there, 
I have just started using anti cheat on my MC server.
It works great, but I have a quick question..
If a player hacks, for example: uses spider, the notification about this player appears for everyone. I was wondering if there was a way so that only admins or a select group can see these notifications?
Thanks,
ncberman96

**Comment by gravitylow** [comment]
It only appears to those with anticheat.system.alert permission.

**Comment by SuperSpyTX** [comment]
@h31ix Do you think I should run through the plugin.yml files and put `default: false` in each permission node?  That way, if the OP is using a faulty plugin and gives out permission nodes by default, that would fix it.

**Comment by gravitylow** [comment]
If the admin is using faulty plugins that give out permission nodes like candy, they've got a much bigger problem on their hands than their players seeing notifications. They should really get to the root of that problem and fix it rather than us trying to cover it up.

**Comment by ncberman96** [comment]
Am I being accused of being a bad admin??
And h3ix, that has not worked for me.. everyone sees it.

**Comment by gravitylow** [comment]
My reply was to @SuperSpyTX not you. In no way am I saying that.
In any case, the plugin uses Bukkit API calls to broadcast to the specific permission; if it's not working as expected it's a problem in the permissions implementation, not AntiCheat itself. Meaning your setup or plugin is not respecting that call or something.

**Comment by thunerb** [comment]
I'm having a similar problem. Prior to updating to 1.4.6 and updating anticheat, players would not receive notifications about their failed checks and despite silent mode being on and them not having the permissions for it, players are receiving these notifications.
As I've not modified my permissions or anything else, are you sure it isn't the plugin having a change that's caused this?

P.s.do note before it DID WORK with my setup until I updated craftbukkit and anticheat

**Comment by SuperSpyTX** [comment]
Please try the latest build and report back with results.

**Comment by thunerb** [comment]
Latest build has reduced these drastically as the fastbreak false positives are almost all fixed.
Player reported receiving the notification when using a diamond efficiency 4 pick.
Players are occasionally receiving other messages but far less frequently now (not to the point they complain a lot of the time)

**Comment by gravitylow** [comment]
There... Were no changes to fast break between 1.4.3 and that build. And the fix is either all or nothing; people won't just be SOMETIMES getting messages

**Comment by SuperSpyTX** [comment]
@h31ix Maybe he was running an outdated build of AntiCheat.

Who knows, this issue seems to be fixed from the comments.

And if this issue still isn't resolved. Please **do not** make another ticket, reply to this one and it'll be reopened.

**Comment by thunerb** [comment]
Ok, replying.
Updated anticheat to the latest version and running craftbukkit beta build 1.4.6 r0.1 (for reference).

After updating to the newest version which fixed players having failed checks for using iron shovels, it's still notifying them when they fail checks (although as most failures were fastbreaks due to false positives, this is happening less frequently).
This includes two checks I have identified so far:
- fly check (which as we have teleport plugins happens a lot - I understand conflicts with other plugins would be difficult to resolve) so I've disabled them being checked for flying.
- One player also had a report of fastbreaking for using diamond pick with efficiency 4, so some players seem to still be receiving the notifications for failing checks.
  These notifications appear solely for the player who failed the check - despite silent mode being turned on.
  I'm unsure if other failed checks appear for the player, but those are the reports I have.

How sure are you silent mode is working properly, as since updating past 1.5.1 seems to have caused a change that no longer makes it work, or work completely correctly based on my experience.
